sdk/template-dpg/Azure.ServiceTemplate.Template/content/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
Do we have any guidance on authoring this file?

Reply:
I am confirm the purpose of this file with Wes and if it is needed for dataplane or not?

Reply:
The file is for deploying live test resources needed for the tests and it will change to bicep file instead of json. So current we will not add it to template, and will change to bicep late when the whole live test framework ready.